web/portal/src/main/webapp/WEB-INF/conf/common/common-configuration.xml

```diff
@@ -178,6 +178,12 @@
 
   <component>
     <type>org.exoplatform.groovyscript.text.TemplateService</type>
+    <init-params>
+      <value-param>
+        <name>templates.collect.statistics</name>
+        <value>${exo.groovy.template.statistics.enable:true}</value>
```

Comment:
I would call it exo.statistics.groovy.template instead, so we could have others exo.statistics... properties and it would make it more consistent. WDYT ?